404.njk

```diff
@@ -0,0 +1,35 @@
+---
+layout: layout.njk
+permalink: 404.html
+title: 404
+eleventyExcludeFromCollections: true
+---
+
+{% import "util/macros.njk" as util %}
+
+<section class="not-found" style="text-align:center;margin: 8em 0 14em;">
+  <h1>40<span style="
+      transform: rotate(337deg) translate(-6px, 4px);
+      display: inline-block;
+  ">4</span></h1>
+  <p>That's a bummer.</p>
+  <p id="not-found-suggestion" hidden></p>
```

Comment:
The paragraphs with just short lines are a bit excessively spaced. I didn't fully test this, but perhaps give it a shot?

```suggestion
  <p>That's a bummer.<br>
  <span id="not-found-suggestion" hidden></span></p>
```

Reply:
Like it a lot 
<img width="448" height="289" alt="image" src="https://github.com/user-attachments/assets/cd3d0ab4-2274-4418-ad5c-046c8cd86179" />

And it also *just* works in the case where we don't have any suggestions.  
<img width="459" height="238" alt="image" src="https://github.com/user-attachments/assets/f8800784-59a2-4388-8605-870270de6c3d" />